packages/elcollectooorr/contracts/artblocks/contract.yaml

```diff
@@ -14,4 +14,8 @@ class_name: ArtBlocksContract
 contract_interface_paths:
   ethereum: build/artblocks.json
 contracts: []
-dependencies: {}
+dependencies:
+  open-aea-ledger-ethereum:
+    version: ==1.22.0
+  web3:
+    version: ==5.25.0
```

Comment:
@angrybayblade I think this is a breaking change. It's probably late for suggestions, but wouldn't it be better to just warn rather than throw an error when doing these checks?